app/models/vacancy.rb

```diff
@@ -87,6 +87,10 @@ def listed?
     published? && expires_at&.future? && (publish_on&.today? || publish_on&.past?)
   end
 
+  def scheduled?
```

Comment:
The scope is called `pending` but it means the same as `scheduled`. We should choose one naming convention to be consistent.